src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -231,27 +362,74 @@ def bpe(self, token):
         self.cache[token] = word
         return word
 
-    # Copied from transformers.models.gpt2.tokenization_gpt2.GPT2Tokenizer.build_inputs_with_special_tokens with GPT2 -> Whisper
-    def build_inputs_with_special_tokens(self, token_ids_0, token_ids_1=None):
-        if self.add_bos_token:
-            bos_token_ids = [self.bos_token_id]
-        else:
-            bos_token_ids = []
+    def set_prefix_tokens(self, language: str = None, task: str = None, predict_timestamps: bool = None):
+        """
+        Override the prefix tokens appended to the start of the label sequence. This method can be used standalone to
+        update the prefix tokens as required when fine-tuning. Example:
 
-        output = bos_token_ids + token_ids_0
+        ```python
+        >>> tokenizer = WhisperTokenizer.from_pretrained("openai/whisper-tiny", language="spanish")
+        >>> tokenizer.set_prefix_tokens(language="french")  # update the language prefix token
```

Comment:
Are we switching from Spanish to Franch here? Would be useful if the comment was clearer on that.

Reply:
Indeed we are! Resolved in: https://github.com/huggingface/transformers/pull/19921/commits/aa8f4cf7c09e60d2ae68b32f58a71213af157c95